src/test/java/mate/academy/bookstore/repository/order/OrderRepositoryTest.java

```diff
@@ -0,0 +1,63 @@
+package mate.academy.bookstore.repository.order;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.List;
+import java.util.Set;
+import mate.academy.bookstore.model.Book;
+import mate.academy.bookstore.model.Order;
+import mate.academy.bookstore.model.OrderItem;
+import mate.academy.bookstore.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.jdbc.Sql;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class OrderRepositoryTest {
+    @Autowired
+    private OrderRepository orderRepository;
+
+    @Test
+    @DisplayName("Verify findByEmail() method works for OrderRepository")
+    @Sql(scripts = {
+            "classpath:database/order/create-order.sql"
+    }, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = {
+            "classpath:database/order/remove-order.sql"
+    }, executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
+    void findByEmail_ValidUserId_returnListOrder() {
+        Order order = new Order()
+                .setId(1L)
```

Comment:
It is difficult to read test when there are such huge order. To improve readability I suggest to move creation of order is separate method. In this case test will contain only 4 lines of code and will be significantly more readable